libraries/lib-audio-io/AudioIO.cpp

```diff
@@ -1139,11 +1152,18 @@ bool AudioIO::AllocateBuffers(
       try
       {
          if( mNumPlaybackChannels > 0 ) {
-            // Allocate output buffers.  For every output track we allocate
-            // a ring buffer of ten seconds
+            // Allocate output buffers.
+            // Allow at least 2x of the buffer latency.
             auto playbackBufferSize =
-               (size_t)lrint(mRate * mPlaybackRingBufferSecs.count());
+               std::max((size_t)lrint(mRate * mPlaybackRingBufferSecs.count()), mHardwarePlaybackLatencyFrames * 2);
```

Comment:
Does this line cause no change on Mac and Windows?

Reply:
I have observed none, both usually report relatively low latency and report it correctly. This specific line cannot really hurt, there is huge inconsistency between `playbackBufferSize`  on different policies and I don't really think that provided ring buffer size matters that much if it is sufficient. 

`mPlaybackQueueMinimum` is different, but it must handle the reported output latency anyway. Simply using some random number is incorrect. Yet again - reported latency is usually lower than the one requested by playback policy. This PR does affect responsiveness of scrubbing on ALSA, but the bug itself makes Audacity **unusable** on ALSA, which is much worse.